docs/index.html

```diff
@@ -0,0 +1,353 @@
+    <link rel="icon" id="icon-32" type="image/png" sizes="32x32" href="./icons/favicon-32x32.png" />
+    <link rel="manifest" href="./site.webmanifest" />
+    <link rel="mask-icon" href="./icons/safari-pinned-tab.svg" color="#5bbad5" />
+    <link rel="shortcut icon" id="icon-ico" href="./icons/favicon.ico" />
+    <meta name="apple-mobile-web-app-title" content="Water.css" />
+    <meta name="application-name" content="Water.css" />
+    <meta name="msapplication-TileColor" content="#00aba9" />
+    <meta name="msapplication-config" content="./icons/browserconfig.xml" />
+    <meta name="theme-color" content="#ffffff" />
+
+    <!-- Startup styles of water.css, so styles don't have to wait until JS is loaded -->
+    <link rel="preload" as="style" href="./water.css/dark.min.css" />
+    <link rel="preload" as="style" href="./water.css/light.min.css" />
+    <link rel="stylesheet" id="js-startup-stylesheet" href="./water.css/water.min.css" />
+
+    <!-- Dynamic version of water.css, overwrites startup styles. JavaScript sets & updates href -->
+    <link rel="stylesheet" id="js-stylesheet" />
+
+    <!-- Custom styles for the documentation / version picker -->
+    <link rel="stylesheet" href="style.css" />
+
+    <meta name="twitter:card" content="summary_large_image" />
+    <meta property="og:type" content="website" />
+    <meta property="og:url" content="https://watercss.kognise.dev/" />
+    <meta property="og:title" content="Water.css" />
+    <meta
+      property="og:description"
+      content="A drop-in collection of CSS styles to make simple websites like this just a little bit nicer."
+    />
+    <meta
+      property="og:image"
+      content="https://raw.githubusercontent.com/kognise/water.css/master/logo.png"
+    />
+    <meta property="og:image:width" content="1154" />
+    <meta property="og:image:height" content="444" />
+    <script type="application/ld+json">
+      {
+        "@context": "http://www.schema.org",
+        "@type": "WebSite",
+        "name": "Water.css",
+        "alternateName": "Water.css",
+        "url": "https://watercss.kognise.dev/"
+      }
+    </script>
+
+    <script>
+      // eslint-disable-next-line no-useless-escape
+      window.navigator.clipboard || document.write('<script src="https://unpkg.com/clipboard-polyfill@2.8.6/dist/clipboard-polyfill.promise.js"><\/script>')
+    </script>
+
+    <!-- Use bright favicons when the browser is in dark mode. -->
+    <script type="module">
+      import faviconSwitcher from 'https://unpkg.com/favicon-mode-switcher@1.0.4/dist/index.min.mjs'
+      faviconSwitcher([
+        { element: '#icon-ico', href: { dark: './icons/light-favicon.ico' } },
+        { element: '#icon-16', href: { dark: './icons/light-favicon-16x16.png' } },
+        { element: '#icon-32', href: { dark: './icons/light-favicon-32x32.png' } }
+      ])
+    </script>
+
+    <script async src="https://www.googletagmanager.com/gtag/js?id=UA-116663597-6"></script>
```

Comment:
Should probably have a privacy policy or some kind of notification on what's collected through Google Tag Manager

Reply:
Or we just remove the tag manager and spare our users from the cookie / privacy popup or bar.
Not like we depend on Analytics to drive business decisions or something

Reply:
The best kind of personal information is the kind you never collect in the first place 😉

Reply:
@kognise

Reply:
I'm gonna remove it altogether, the account isn't even active lmao. Also sorry! For some reason GitHub wasn't showing me these so I thought nothing was happening - my bad for not double checking.

Reply:
Boom 882e01f17c97a101a8a2cd877097ac09fb260600